src/main/java/org/osiam/client/OsiamConnector.java

```diff
@@ -62,29 +60,25 @@
  */
 public class OsiamConnector {
 
-    public static final int DEFAULT_CONNECT_TIMEOUT = 2500;
-    public static final int DEFAULT_READ_TIMEOUT = 5000;
+    static final int DEFAULT_CONNECT_TIMEOUT = 2500;
+    static final int DEFAULT_READ_TIMEOUT = 5000;
     private static final int DEFAULT_MAX_CONNECTIONS = 40;
-    private static final int DEFAULT_MAX_CONNECTIONS_PER_ROUTE = 20;
 
-    private static final PoolingHttpClientConnectionManager CONNECTION_MANAGER;
-    private static final javax.ws.rs.client.Client client;
     public static final ObjectMapper objectMapper = new ObjectMapper();
 
-    static {
-        CONNECTION_MANAGER = new PoolingHttpClientConnectionManager();
-        CONNECTION_MANAGER.setMaxTotal(DEFAULT_MAX_CONNECTIONS);
-        CONNECTION_MANAGER.setDefaultMaxPerRoute(DEFAULT_MAX_CONNECTIONS_PER_ROUTE);
+    private static final PoolingHttpClientConnectionManager CONNECTION_MANAGER =

```

Comment:
It actually is, it's only used as a reference and never together with the dot operator.